tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/failover/BackupSyncJournalTest.java

```diff
@@ -154,14 +154,9 @@ public void testReserveFileIdValuesOnBackup() throws Exception {
 
       assertEquals(size, backupMsgJournal.getFileSize(), "file sizes must be the same");
       Set<Pair<Long, Integer>> backupIds = getFileIds(backupMsgJournal);
-
-      int total = 0;
       for (Pair<Long, Integer> pair : primaryIds) {
-         total += pair.getB();
```

Comment:
Not sure we need empty loop this leaves, both here and the similar one that follows immediately.

I would instead wonder what they originally existed to do and whether the test simply isnt/wasnt checking something that it was actually meant to.

Reply:
Fixed.

Reply:
That further change, to remove the loops, then just leaves both of the collections being looped over as unused themselves.

The test is called testReserveFileIdValuesOnBackup, and the collections are seemingly about the Ids. If its not using them then it doesnt seem like its actually testing what it meant to. If these bits are actually removed, its not clear the test shouldnt just be deleted instead.

Reply:
@clebertsuconic  are you clear on what the test was maybe originally trying to do, and if it can/should be fixed?

Reply:
The assertion related to these loops was removed in af9bd7b84aad32e4fe30f2c8909e51cf7300b475 by @clebertsuconic. The test was originally added way back in HornetQ by Francisco Borges (who implemented replication initially).

Reply:
The assertion related to these loops was removed in af9bd7b84aad32e4fe30f2c8909e51cf7300b475 by @clebertsuconic. The test was originally added way back in HornetQ by Francisco Borges (who implemented replication initially).

Reply:
Yeah, I don't actually know what it is validating.

Reply:
> That further change, to remove the loops, then just leaves both of the collections being looped over as unused themselves.
> 

Also, if we remove creation of the collections and leave the rest of the test, the getFileIds and getPair methods used for creating them also become unused as well.

Reply:
I decided to keep the test and just clean it up by removing the unnecessary variables, loops, methods, etc.